src/commands/schema/diff.ts

```diff
@@ -1,17 +1,20 @@
-const SchemaCommand = require("../../lib/schema-command.js").default;
-const fetch = require("node-fetch");
+import SchemaCommand from "../../lib/schema-command";
+import fetch from "node-fetch";
 
-class DiffSchemaCommand extends SchemaCommand {
+export default class DiffSchemaCommand extends SchemaCommand {
   static flags = {
     ...SchemaCommand.flags,
   };
 
+  static description = "Print the diff between local and remote schema.";
+  static examples = ["$ fauna schema diff --dir schemas/myschema"];
```

Comment:
should we show an example without the --dir as well?

Reply:
yeah we should